### PR TITLE
Add convenience init to avoid itemSize and itemsPerRow confusion

### PR DESCRIPTION
**Pull request by zenangst**
Fixes https://github.com/zenangst/Blueprints/issues/43

### CODE REVIEW

File: Sources/Shared/Core/VerticalBlueprintLayout.swift [hunk 1]
```diff
@@ -46,6 +45,66 @@
     self.scrollDirection = .vertical
   }
 
+  /// An initialized vertical collection view layout object.
+  ///
+  /// - Parameters:
+  ///   - itemsPerRow: The amount of items that should appear on each row.
+  ///   - estimatedItemSize: The estimated size to use for cells.
```
Comment (christoff-1992) [inline Sources/Shared/Core/VerticalBlueprintLayout.swift]: Does this need removing? :)

Comment (zenangst) [reply Sources/Shared/Core/VerticalBlueprintLayout.swift]: Nice catch!

File: Sources/Shared/Core/VerticalBlueprintLayout.swift [hunk 2]
```diff
@@ -46,6 +45,66 @@
...
+              stickyHeaders: stickyHeaders,
+              stickyFooters: stickyFooters,
+              animator: animator)
+  }
+
+  /// An initialized vertical collection view layout object.
+  ///
+  /// - Parameters:
+  ///   - itemSize: The default size to use for cells.
+  ///   - estimatedItemSize: The estimated size to use for cells.
```
Comment (christoff-1992) [inline Sources/Shared/Core/VerticalBlueprintLayout.swift]: Again here

**Comment by zenangst** [comment]
@christoff-1992 looks good! :)

**Pull request merged**